From 0e64ed958ca08f887f73db0bf472ecc851ebd82b Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Tue, 13 Jun 2023 15:55:49 +0200
Subject: [PATCH 5/5] Link to abseil logging libraries

---
 third_party/systemlibs/protobuf.BUILD | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/third_party/systemlibs/protobuf.BUILD b/third_party/systemlibs/protobuf.BUILD
index 3e42a1b..7dd076c 100644
--- a/third_party/systemlibs/protobuf.BUILD
+++ b/third_party/systemlibs/protobuf.BUILD
@@ -62,19 +62,19 @@ genrule(
 
 cc_library(
     name = "protobuf",
-    linkopts = ["-lprotobuf"],
+    linkopts = ["-lprotobuf", "-labsl_log_internal_check_op", "-labsl_log_internal_message"],
     visibility = ["//visibility:public"],
 )
 
 cc_library(
     name = "protobuf_headers",
-    linkopts = ["-lprotobuf"],
+    linkopts = ["-lprotobuf", "-labsl_log_internal_check_op", "-labsl_log_internal_message"],
     visibility = ["//visibility:public"],
 )
 
 cc_library(
     name = "protoc_lib",
-    linkopts = ["-lprotoc"],
+    linkopts = ["-lprotoc", "-labsl_log_internal_check_op", "-labsl_log_internal_message"],
     visibility = ["//visibility:public"],
 )
 
-- 
2.39.2 (Apple Git-143)

